test-services/splunk/install-app.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+ACCESS_TOKEN=${ACCESS_TOKEN:-testing123}
+INGEST_HOST=${INGEST_HOST:-https://ingest.us0.signalfx.com}
+
+/opt/splunk/bin/splunk list user -auth admin:testing123
+/opt/splunk/bin/splunk add monitor /opt/splunk/var/log/splunk/splunkd.log
+/opt/splunk/bin/splunk install app /code/splunk-forwarder/signalfx-forwarder-app-*.tar.gz
```

Comment:
Add some  whitespace between some of these lined based on some logical grouping.